test/integration/local/test_mnist_training.py

```diff
@@ -38,13 +39,22 @@ def test_mnist_training_and_serving(docker_image, sagemaker_local_session, local
     with local_mode.lock():
         try:
             model = mx.create_model(model_server_workers=NUM_MODEL_SERVER_WORKERS)
-            predictor = model.deploy(1, local_instance_type)
+            predictor = _csv_predictor(model, local_instance_type)
             data = numpy.zeros(shape=(1, 1, 28, 28))
             predictor.predict(data)
         finally:
             mx.delete_endpoint()
 
 
+def _csv_predictor(model, instance_type):
+    predictor = model.deploy(1, instance_type)
+    predictor.content_type = 'text/csv'
+    predictor.serializer = csv_serializer
+    predictor.accept = 'text/csv'
+    predictor.deserializer = None
```

Comment:
why don't we test the deserializer here as well?

Reply:
these are Python SDK entities; the Python SDK doesn't have a built-in CSV deserializer